requirements/serve.txt

```diff
@@ -1,3 +1,6 @@
 gradio
 protobuf
 tritonclient[grpc]
+timm
```

Comment:
Please don't add them in serve.txt. 
They are dependencies of VLMs models, not LLMs. What's more, they are not common dependencies among VLM models.